assets/js/components/DashboardNavigation.stories.js

```diff
@@ -19,13 +19,160 @@
 /**
  * Internal dependencies
  */
+
 import DashboardNavigation from './DashboardNavigation';
+import { Provider as ViewContextProvider } from './Root/ViewContextContext';
+import { CORE_WIDGETS } from '../googlesitekit/widgets/datastore/constants';
+import {
+	CONTEXT_MAIN_DASHBOARD_TRAFFIC,
+	CONTEXT_MAIN_DASHBOARD_CONTENT,
+	CONTEXT_MAIN_DASHBOARD_SPEED,
+	CONTEXT_MAIN_DASHBOARD_MONETIZATION,
+} from '../googlesitekit/widgets/default-contexts';
+import WithRegistrySetup from '../../../tests/js/WithRegistrySetup';
+import { VIEW_CONTEXT_DASHBOARD } from '../googlesitekit/constants';
 
-const Template = ( args ) => <DashboardNavigation { ...args } />;
+const Template = ( { setupRegistry, viewContext, ...args } ) => (
+	<WithRegistrySetup func={ setupRegistry }>
+		<ViewContextProvider value={ viewContext }>
+			<DashboardNavigation { ...args } />
+		</ViewContextProvider>
+	</WithRegistrySetup>
+);
 
 export const DefaultDashboardNavigation = Template.bind( {} );
-DashboardNavigation.storyName = 'DashboardNavigation Default State';
-DashboardNavigation.args = {};
+DefaultDashboardNavigation.storyName = 'DashboardNavigation Default State';
```

Comment:
No need to repeat the `Dashboardnavigation` group name in every story title. Just use the `Default State` name for this story and `Monetization Hidden State` for the other story.